File: iterator/python/implementations/physical_book.py
from interfaces.book import Book

class PhysicalBook(Book):
    def __init__(self, title: str, author: str):
        self._title = title
        self._author = author

    def get_title(self) -> str:
        return self._title

    def get_author(self) -> str:
        return self._author

File: iterator/java/implementations/PhysicalBook.java
package iterator.java.implementations;

import iterator.java.interfaces.Book;

public class PhysicalBook implements Book {
    private String title;
    private String author;

    public PhysicalBook(String title, String author) {
        this.title = title;
        this.author = author;
    }

    @Override
    public String getTitle() {
        return title;
    }

    @Override
    public String getAuthor() {
        return author;
    }
}

